src/pages/inputPage.js

```diff
@@ -1,8 +1,112 @@
-import React from 'react';
-import { StyleSheet, Text } from 'react-native';
+import React, {useRef} from 'react';
+import * as Theme from "@assets/theme.json";
+import {StyleSheet, Text, TextInput, View, ScrollView, Pressable} from 'react-native';
+import Button from "components/button";
+import DatePicker from 'react-native-date-picker'
+
 
 const InputPage = () => {
-  return <Text>input</Text>;
+  const refWeight = useRef();
+  const refFat = useRef();
+  return (
+    <ScrollView style={styles.container}>
+      <DateBox name="Date:"/>
+      <FloatingBox name="Weight:" suffix="kg" maxNum="200" refMain={refWeight} refSubmit={refFat}/>
+      <FloatingBox name="Fat:" suffix="%" maxNum="100" refMain={refFat} refSubmit={null}/>
+      <InputPageFotter />
+    </ScrollView>
+  );
+}
+
+const InputPageFotter = () => {
+  return (
+    <View style={styles.buttonGroup}>
+      <Button title="Done" icon="done-all-outline"/>
+      <Button title="Add" icon="plus-square-outline"/>
+    </View>
+  )
 }
 
+const DateBox = (pros) => {
+  return (
+    <View>
+
+    </View>
+  );
+};
+
+const FloatingBox = (pros) => {
+  const [number, onChangeNumber] = React.useState(null);
```

Comment:
suggestion: use `setNumber` instead of `onChangeNumber` to comply with React's convention.